tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -47,10 +52,40 @@ def _gen_kernel_cubin_impl(ctx):
         cubins.append(cubin)
     return [CubinInfo(cubins = cubins)]
 
+def _gen_kernel_cubin_impl_rocm(ctx):
```

Comment:
I guess this could be shared between both cuda and rocm, if we move the line
```
arch = arch.replace("compute_", "sm_")
```
to the cuda-specific calling function.

Reply:
correct